### ISSUE TITLE
SELECT/INSERT/UPDATE/DELETE over REST

### ISSUE THREAD

**Issue by dedok** [tarantool/nginx_upstream_module#66]
I proposing a configuration for this issue

``` nginx.conf
{
 # Contexts location, location if
 location = /my_insert {
   tnt_http_insert [on, off];
   tnt_pass UPSTREAM;
 }
 location = /my_delete {
   tnt_http_delete [on, off];
   tnt_pass UPSTREAM;
 }
 ... and so on ...
} 
```

**Comment by dedok** [comment]
I need think about the issue...

**Comment by dedok** [comment]
Added: replace, delete, insert, select in b108c1ea8709adf36664c75028db794e1ffbf336

**Comment by morokin** [comment]
Can you please add description of HTTP status codes returning by upstream module to documentation.

Thanks

**Comment by dedok** [comment]
Fixed in db9356ef8bb6b6f30b5f2cf0881248a008a6ac50
